cmd/nmi/main.go

```diff
@@ -101,3 +107,20 @@ func main() {
 		klog.Fatalf("%s", err)
 	}
 }
+
+func getTokenClient(client k8s.Client) nmi.TokenClient {
+	klog.Infof("Initializing in %s mode", strings.ToLower(*operationMode))
```

Comment:
Can we please perform a ToLower once at the beginning as soon as we get the parameters ?